wiki/en/Custom-Directories.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: wiki
+title: "Custom Directories"
+lang: "en"
+permalink: "/wiki/Custom-Directories"
+---
+
+{% include breadcrumb.html root="More" branch1="Server Administration" branch1-url="Running-a-Server" %}
+
+
+# Running a Custom Directory 
+
+This is a specialised Jamulus Server configuration, as described in [Server Types](Running-a-Server#server-types). 
+
+To view Servers listed by a Custom Directory, users must enter the address of that Directory in their client's Settings > Advanced Setup > Custom Directories field. Multiple addresses can be added in this way if needed. Custom Directories will then appear in the Directory drop-down list on their Connect window. Custom Directories otherwise work for clients in the same way as Public Directories, displaying a list of Servers registered with them.
+
+To run a Server as a Directory, it should be started with the `--directoryserver` option to make itself (that is, 'localhost' or '127.0.0.1') the Directory to query for Servers.
+
+When running a Server with the GUI, select "Custom" from the Directory drop-down list, and set the Custom Directory Address field in the Options tab.  
```

Comment:
Better done the other way around -- set the Custom Directory server address to "localhost" then switch to "Custom".  Just in case there's something in the Custom Directory server address already.

Reply:
Like this? "When running a Server with the GUI, set the Custom Directory server address in the Options tab to "localhost", then select "Custom" from the Directory drop-down list."

Reply:
Yep :)